src/glass/src/app/shared/services/api/services.service.ts

```diff
@@ -116,6 +131,16 @@ export class ServicesService {
     return this.http.get<ServiceDesc[]>(`${this.url}/`);
   }
 
+  public validateServiceName(service_name: string): Observable<boolean> {
```

Comment:
What about using this name? Looks clearer to me.
```suggestion
  public exists(service_name: string): Observable<boolean> {
```

Reply:
Yes, fine with me